src/apiary.apib

```diff
@@ -892,6 +896,23 @@ View all Campaigns that the Publisher is associated with, along with the status
     + Attributes
         + campaigns (array[Publisher Campaign Wrapper])
 
+## Publisher Campaign Tracking [/user/publisher/{publisher_id}/campaign/{status}/tracking]
+
++ Parameters
+    + publisher_id: 1l1007802 - ID of the publisher
+    + status: (enum) - The status of the publisher on the campaign
```

Comment:
Correct. The docs show the absence of (required) defaults to [required](https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#description-13). I have added the data type for specifics now.